### PR TITLE
correcting json fields

### PR DESCRIPTION
**Pull request by mahikaw**
Adressess #142

### CODE REVIEW

File: sample/FOSSASIA15/events [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
-    "description": "FOSSASIA is the premier Free and Open Source technology event in Asia for developers, start-ups, and contributors. Projects at FOSSASIA range from open hardware, to design, graphics and software. FOSSASIA was established in 2009. Previous events took place in Cambodia and Vietnam.", 
+    "description": "The Smart City with Open Source, Big Data, Design Thinking and Free Knowledge Tools at Asia's Premier 2015 Open Technology Event", 
+    "social_links", 
     "background_image": null, 
     "logo": "img/fossasia-long.png", 
     "organizer_name": "FOSSASIA", 
-    "organizer_description": "The Smart City with Open Source, Big Data, Design Thinking and Free Knowledge Tools at Asia's Premier 2015 Open Technology Event", 
-    "social_links": [
+    "organizer_description": "FOSSASIA is the premier Free and Open Source technology event in Asia for developers, start-ups, and contributors. Projects at FOSSASIA range from open hardware, to design, graphics and software. FOSSASIA was established in 2009. Previous events took place in Cambodia and Vietnam.": [
         {
```
Comment (heysadboy) [inline sample/FOSSASIA15/events]: @mahikaw I think you forgot to put "social_links" tag here after "organizer_description". Starting array directly?

**Comment by mahikaw** [comment]
@mariobehling @niranjan94 Please review and merge it.

**Comment by mariobehling** [comment]
@mahikaw It seems the review process is still not clear. There need to be usually three co-developers who approve your PR before you should ask mentors to look over them. Sometimes also two other contributors are ok, but for the beginning three are expected.

Why are there so many PRs open and not getting finished?

**Comment by mariobehling** [comment]
Please actively ask your co-developers to review PRs and ping them here in the PR and on the chat.

**Comment by mahikaw** [comment]
@arp95 @shailesh351 @shreyasnbhat please review

**Pull request merged**